providers/dns/gcore/internal/client.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/go-acme/lego/v4/providers/dns/internal/errutils"
 )
 
-const defaultBaseURL = "https://api.gcorelabs.com/dns"
```

Comment:
Due to make a full transition to the gcore.com domain, we are ending support for api.gcorelabs.com host and related endpoints.

https://status.gcore.com/incidents/cg0694b1q3tr